### DISCUSSION TITLE
[FR] Sort and filter licenses by columns

### DISCUSSION THREAD

**Discussion by Kashkin** [craftcms/console#89] [Craft Console]
It would be helpful in managing licenses if we could sort by the columns in the license table (License Key | Edition | Domain).

**Comment by timkelty** [comment]
Also, from https://github.com/craftcms/console/issues/50
> It would be great if there was a way to filter by expired/expiring/active states and/or a date selection.

**Comment by mediagirl** [comment]
+1 for sortable columns please.

Also, what is the default sorting column? My list of licenses doesn't appear to sort NEWEST to OLDEST.

**Comment by timkelty** [comment]
Good question @mediagirl… 😬 
They should always be returned by date created/descending now.

**Comment by mediagirl** [comment]
Much better. Thank you!
